nasa-gibs/worldview
# WV-3297 Replace CMR with DescribeDomains for DataAvailability

## Description

> Investigate/Implement - Replace CMR with DescribeDomains for data availability bar(s)

## How To Test

1. `git checkout WV-3297-replace-cmr-with-describedomains`
2. `npm run build && npm run watch`
3. Open this [link](http://localhost:3000/?v=-172.665657727155,-20.930301829778642,-5.4153650290339215,75.08375508951309&z=4&ics=true&ici=5&icd=6&l=Coastlines_15m,TEMPO_L2_Formaldehyde_Vertical_Column_Granule(count=1)&lg=true&t=2024-08-24-T16%3A48%3A00Z)
4. The TEMPO layer should load faster and the data availability bars should have similar functionality when compared to prod.

## PR Submission Checklist

This is simply a reminder of what we are going to look for before merging your code.

- I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/main/.github/CONTRIBUTING.md) doc
- I have added necessary documentation (if applicable)
- I have added tests that prove my fix is effective or that my feature works (if applicable)
- Any dependent changes have been merged and published in downstream modules (if applicable)

## Merging

Please use the `squash and merge` commit method unless each commit in your branch is vital to the commit history of main.

@nasa-gibs/worldview
